.github/workflows/ci.yml

```diff
@@ -458,6 +458,9 @@ jobs:
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $RETRY $MAVEN clean install ${MAVEN_FAST_INSTALL} ${MAVEN_GIB} -am -pl "${{ matrix.modules }}"
+      - name: Prefetch Ryuk
+        run: |
+          $RETRY docker pull $(cat testing/trino-testing-containers/src/main/resources/io/trino/testing/containers/testcontainers-ryuk-version.txt)
```

Comment:
```suggestion
          $RETRY docker pull testcontainers/ryuk:$(cat testing/trino-testing-containers/src/main/resources/io/trino/testing/containers/testcontainers-ryuk-version.txt)
```

Reply:
i put that thing in the resource file, so it's tested along with the version